pandas/tests/series/test_dtypes.py

```diff
@@ -506,6 +506,33 @@ def test_is_homogeneous_type(self):
         assert Series([1, 2])._is_homogeneous_type
         assert Series(pd.Categorical([1, 2]))._is_homogeneous_type
 
+    @pytest.mark.parametrize(
```

Comment:
this should go in pandas/tests/test_algos.py.

Reply:
Thanks, @jreback - moved the test there.